src/popup/components/Toast/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from "react";
```

Comment:
toast notifications are a v small use case right now. Maybe at some pt it's worth replacing this with a lib - but let's not do that til we actually think it's necessary